mist-lib/src/db/migrations/0005_genomes-taxonomy_indices.sql

```diff
@@ -0,0 +1,18 @@
+CREATE INDEX trgm_idx_genomes_name ON genomes USING gin (name gin_trgm_ops);
+CREATE INDEX trgm_idx_genomes_superkingdom ON genomes USING gin (superkingdom gin_trgm_ops);
+CREATE INDEX trgm_idx_genomes_phylum ON genomes USING gin (phylum gin_trgm_ops);
+CREATE INDEX trgm_idx_genomes_class ON genomes USING gin (class gin_trgm_ops);
+CREATE INDEX trgm_idx_genomes_orderr ON genomes USING gin (orderr gin_trgm_ops);
+CREATE INDEX trgm_idx_genomes_family ON genomes USING gin (family gin_trgm_ops);
+CREATE INDEX trgm_idx_genomes_genus ON genomes USING gin (genus gin_trgm_ops);
+CREATE INDEX trgm_idx_genomes_assemblylevel ON genomes USING gin (assembly_level gin_trgm_ops);
```

Comment:
I'm a stickler for consistency (reduces mental burden between developers when things are done in the same style). Given that all other SQL migrations have keywords in lowercase, please lowercase all the keywords here too (e.g. `create index ...`). Again for consistency, please name `trgm_idx_genomes_assembly_level`.

Reply:
OK.

Reply:
Will do.